pyproject.toml

```diff
@@ -3,7 +3,7 @@ line-length = 100
 
 [tool.poetry]
 name = "dbt-copilot-tools"
-version = "3.1.2"
+version = "3.1.3"
```

Comment:
This should only change when we decide to do a release, which we probably should once this is merged.